src/components/app.tsx

```diff
@@ -151,16 +139,12 @@ export const authAndConnect = (stores: IStores, onQAClear?: (result: boolean, er
 export class AppComponent extends BaseComponent<IProps, IState> {
 
   public state: IState = {
-    qaCleared: false,
-    qaClearError: undefined
   };
```

Comment:
Is there a reason to keep `IState` as an empty object?

Reply:
It should work fine without it. I'll remove it.